components/cloud-foundry/frontend/src/view/applications/workflows/add-service-workflow/instance.html

```diff
@@ -54,6 +54,32 @@
           </button>
         </td>
       </table>
+
+      <div class="form-group form-json-editor-input"
+        ng-class="{'has-error': addInstanceForm.createNewForm.name.$invalid && addInstanceForm.createNewForm.name.$dirty}">
+        <label class="control-label" required translate>
+          app.app-info.app-tabs.services.create.params-label
+        </label>
+        <a href="#" class="input-box-edit" ng-click="wizardCtrl.options.editParams = true; wizardCtrl.options.hideFooterButtons = true" translate>app.app-info.app-tabs.services.create.edit-params</a>
+        <div class="json-edit-box">{{ wizardCtrl.options.userInput.params | jsonString }}</div>
+      </div>
+  
     </form>
   </div>
 </div>
+
+<!-- Parameter editor -->
+<div ng-if="wizardCtrl.options.editParams" class="async-dialog-param-editor" ng-class="{'param-editor-open' : wizardCtrl.options.editParams}">
+  <form name="form.paramEditor" class="async-dialog-param-editor param-editor-container">
+      <div class="form-group json-input-field" focusable-input ng-model-options="{ debounce: 200 }" ng-class="{'has-error': form.paramEditor.$invalid && form.paramEditor.$dirty}">
+        <label class="control-label" translate>app.app-info.app-tabs.services.create.service-params</label>
```

Comment:
Sorry, late comment... instead of 'Parameters' could we call this 'Binding Parameters' and in the create instance dialog 'Instance Parameters'?